### ISSUE TITLE
toc

### ISSUE THREAD

**Issue by 1233fsfgr** [michele-segata/plexe-pyapi#17]
Hello, developer, I want to let the team vehicles take over. I can't take over successfully using Traci.toc settings. How should it be implemented?

**Comment by michele-segata** [comment]
By take over, do you mean overtake?

**Comment by 1233fsfgr** [comment]
No,I mean take over control (TOC) that the vehicle changes from automatic driving to manual driving.

**Comment by 1233fsfgr** [comment]
Ok, thank you very much for responding to my question. Thank you again.
